Glimmer/Assets/Scripts/EnemyController.cs

```diff
@@ -6,13 +6,12 @@ public class EnemyController : MonoBehaviour
 {
     private void OnTriggerEnter(Collider other)
     {
-        PlayerController playerController = other.gameObject.GetComponent<PlayerController>();
+        DetectionAura playerController = other.gameObject.GetComponent<DetectionAura>();
```

Comment:
This is very wrong... why a DectectionAura typed reference is called "playerController"? Stop copy pasting code please